### DISCUSSION TITLE
Hiding a button based off anther buttons state.

### DISCUSSION THREAD

**Discussion by Schoultzies** [jcwillox/lovelace-paper-buttons-row#88] [General]
Firstly,- paper-buttons-row is AWESOME!!!  I am asking if (on a particular row) a button could be shown/hidden according to a particular state? I am trying to create an alarm panel where on the top row, there are two buttons, 'Arm Home' and 'Arm Away'. I would very much like to have a situation whereby arming either option, replaces those two buttons on the row with a singe button showing the state, 'Armed Home' or 'Armed Away'. Possible? 
![aaaa](https://user-images.githubusercontent.com/61315187/166463441-862a8f0e-97f7-4eec-910e-68f4a203bbe1.JPG)

**Comment by maxshcherbina** [comment]
Take a look at this discussion! Just a few posts down ;) 

https://github.com/jcwillox/lovelace-paper-buttons-row/discussions/65
